### ISSUE TITLE
🐞 default version is too old 

### ISSUE THREAD

**Issue by levlaz** [dagger/dagger-for-github#168]
### What is the issue?

If I don't explicitly set the version number, it seems to want to use 0.13.7 for some reason, can we make this latest somehow?

https://github.com/levlaz/snippetbox/actions/runs/12521681484/job/34929042207 

### Dagger version

dagger v0.15.1 (registry.dagger.io/engine:v0.15.1) darwin/arm64

**Comment by jedevc** [comment]
The default version to pull is based on the version of the action - if you pin the action to `v6.15.1`, you always get the same version of dagger: `v0.13.7`.

You can specify to always get the latest version, by using the `version` arg documented in the README.

**Comment by sw360cab** [comment]
@jedevc in case of a local module call, is there a way to tell an action to use the same version of the module itself?
It happened to me different failures of actions because of a simple mismatch between Dagger version in local module (likely updated), and the Dagger version in the github action (forgot to update/sync version).

IMHO, It should be possible to instruct the actions to "use the same version of what is in `dagger.json`"
What do you think/suggest?

**Comment by jedevc** [comment]
@sw360cab, yup that sgtm - I'd like that feature. See https://github.com/dagger/dagger-for-github/issues/138, where it was also proposed.

**Comment by jedevc** [comment]
Also gonna close this, from `8.0.0` the version is now *not* tied to the dagger version - it's not required as an explicit parameter.
